src/sagemaker/clarify.py

```diff
@@ -393,6 +431,94 @@ def to_dict(self) -> Dict[str, Any]:  # pragma: no cover
         return segment_config_dict
 
 
+class TimeSeriesDataConfig:
+    """Config object for TimeSeries explainability specific data fields."""
+
+    def __init__(
+        self,
+        target_time_series: Union[str, int],
+        item_id: Union[str, int],
+        timestamp: Union[str, int],
+        related_time_series: Optional[List[Union[str, int]]] = None,
+        item_metadata: Optional[List[Union[str, int]]] = None,
+    ):
+        """Initialises TimeSeries explainability data configuration fields.
+
+        Args: #TODO: verify param descriptions are accurate
```

Comment:
please, review the docs with science team

Reply:
Will do before finalisation.